docs/migration-progress.md

```diff
@@ -0,0 +1,40 @@
+# Migration progress
+
+UCX tracks migration progress of _business_ resources: workspace resources that contribute to business value.
+(The term "business resource" comes from the UCX team and is **not** Databricks terminology.) We identified the
+following business resources:
```

Comment:
Clarify that these are only the resources that are not currently tracked. In particular I think we also have:

 - Tables
 - UDFs
 - Clusters
 - Cluster Policies

I think these are already covered; this document is limited to remaining work.